services/beaconchain/src/main/java/tech/pegasys/teku/services/beaconchain/BeaconChainController.java

```diff
@@ -807,6 +808,7 @@ protected void initP2PNetwork() {
             .eventChannels(eventChannels)
             .recentChainData(recentChainData)
             .gossipedBlockProcessor(blockManager::validateAndImportBlock)
+            .gossipedBlockAndBlobsProcessor(OperationProcessor.noop())
```

Comment:
Is this correct? Don't we do anything with the BlockAndBlob messages we receive?

Reply:
yes I did it on purpose. I want to modify `BlockManager` and `BlockValidator`  (and maybe `BlockImporter`) in a dedicated PR